src/languages/es.js

```diff
@@ -260,6 +261,22 @@ export default {
     },
     paymentsPage: {
         paymentMethodsTitle: 'Métodos de pago',
+        allSet: 'Todo listo!',
+        transferConfirmText: ({amount}) => `${amount} llegará a tu cuenta en breve!`,
+        gotIt: 'Gracias!',
+    },
+    transferAmountPage: {
+        transfer: ({amount}) => `Transferir${amount ? (` ${amount}`) : ''}`,
```

Comment:
Curious what the extra parenthesis around ```(` ${amount}`)``` is for? 🤔

Reply:
It is added automatically by prettier.